TopsortAnalytics/src/main/java/com/topsort/analytics/banners/BannerView.kt

```diff
@@ -0,0 +1,36 @@
+package com.topsort.analytics.banners
+
+import android.content.Context
+import android.widget.ImageView
+import coil.load
+import com.topsort.analytics.Analytics
+import com.topsort.analytics.model.Placement
+import com.topsort.analytics.model.auctions.EntityType
+
+class BannerView(
+    context: Context,
+    config: BannerConfig,
+    path: String,
+    location: String,
+    clickCallback: (String, EntityType) -> Unit
+) : ImageView(context) {
+    init {
+        val result = runBannerAuction(config)
```

Comment:
can this function throw? I'm guessing the `null` value is when there are no winners, right?

Reply:
Yes, `null` is for no winners. I am not sure if the function can throw downstream, though. @anonvt can we throw on the auction client?

Reply:
Don't know if kotlin allow putting `throws` like java, to add the error to the type signature of the function

Reply:
imo the client object should throw, but the view should be able to handle errors